rasa/core/training/visualization.py

```diff
@@ -359,9 +359,26 @@ def _create_graph(fontsize: int = 12) -> "networkx.MultiDiGraph":
     return graph
 
 
-def sanitize(s):
+def escape(pattern: Text):
+    """Escape special characters in a string.
+
+    Used to fix different behavior in re.escape() for python
+    versions < 3.7 compared to >= 3.7.
+    """
+    import sys
+    import re
+
+    _special_chars_map = {i: "\\" + chr(i) for i in b"()[]{}?*+-|^$\\.&~# \t\n\r\v\f"}
+
+    if sys.version_info >= (3, 7):
```

Comment:
please add a comment why you are doing this

Reply:
it explains in the docstring for the function